browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -95,6 +96,24 @@ class MarkdownNoteDetail extends React.Component {
     return title
   }
 
+  getPercentageOfCompleteTodo (value) {
+    let splitted = value.split('\n')
+    let numberOfTodo = 0
+    let numberOfCompletedTodo = 0
+
+    for (let i = 0; i < splitted.length; i++) {
+      let trimmedLine = splitted[i].trim()
+      if (trimmedLine.match(/^- \[\s|x\] ./)) {
```

Comment:
Thank you for telling me. Oh... I had overlooked + and *. I'll add these operator.